builder/vsphere/driver/vm.go

```diff
@@ -580,6 +581,30 @@ func (vm *VirtualMachineDriver) Configure(config *HardwareConfig) error {
 		}
 		confSpec.DeviceChange = append(confSpec.DeviceChange, spec)
 	}
+
+	if config.Displays != 0 {
+		devices, err := vm.vm.Device(vm.driver.ctx)
+		if err != nil {
+			return err
+		}
+
+		l := devices.SelectByType((*types.VirtualMachineVideoCard)(nil))
+		if len(l) != 1 {
+			return err
+		}
+		card := l[0].(*types.VirtualMachineVideoCard)
+
+		// The amount of video memory must be set along with the number of displays in the same config spec
+		card.VideoRamSizeInKB = config.VideoRAM
```

Comment:
I tried something similar to this during my testing and had issues but let me give this another try to see if the code could be cleaner!

Reply:
Okay let me know if you are able to get it working. There is no issue with the duplicate I just thought to ask since ViedeoRam is required and there is only a one line difference. I'll go with whatever option you find works best.

Reply:
Simplified to the following:

```go
	if config.Displays == 0 {
		config.Displays = 1
	}

	if config.VideoRAM != 0 || config.Displays != 0 {
		devices, err := vm.vm.Device(vm.driver.ctx)
		if err != nil {
			return err
		}
		l := devices.SelectByType((*types.VirtualMachineVideoCard)(nil))
		if len(l) != 1 {
			return err
		}
		card := l[0].(*types.VirtualMachineVideoCard)
		card.VideoRamSizeInKB = config.VideoRAM
		card.NumDisplays = config.Displays
		spec := &types.VirtualDeviceConfigSpec{
			Device:    card,
			Operation: types.VirtualDeviceConfigSpecOperationEdit,
		}
		confSpec.DeviceChange = append(confSpec.DeviceChange, spec)
	}
```